reana/reana_dev/utils.py

```diff
@@ -335,3 +344,147 @@ def get_prefixed_component_name(component):
     :return: Prefixed name.
     """
     return "-".join([INSTANCE_NAME, component])
+
+
+def get_component_version_files(component):
+    """Get a dictionary with all component's version files."""
+    version_files = {}
+    for file_ in [
+        HELM_VERSION_FILE,
+        OPENAPI_VERSION_FILE,
+        JAVASCRIPT_VERSION_FILE,
+        PYTHON_VERSION_FILE,
+    ]:
+        file_relative_path = run_command(
+            f"git ls-files | grep -w {file_} || true",
+            component,
+            display=False,
+            return_output=True,
+        )
+        if file_relative_path:
+            version_files[file_] = os.path.join(
+                get_srcdir(component=component), file_relative_path
+            )
+
+    return version_files
+
+
+def get_component_current_version_from_file(component):
```

Comment:
Naming comment: `..._from_file()` evokes a feeling that it is getting version from one file, so I was expecting to see it as argument.  Perhaps rename to `get_current_component_version_from_source_files()` or somesuch.